src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
+        for (Machine i : machines) {
+            i.doWork();
+            i.stopWork();
+        }
```

Comment:
better not to name variables (excluding counters etc) with single letter
```suggestion
        for (Machine machine : machines) {
            machine.doWork();
            machine.stopWork();
        }
```